tests/acceptance/bootstrap/WebDav.php

```diff
@@ -2673,6 +2673,31 @@ public function userDeletesFileFromSpaceUsingFileIdPath(string $user, string $fi
 		$this->pushToLastStatusCodesArrays();
 	}
 
+	/**
+	 * @Given user :user has updated file with content :content using file-id :fileId
+	 *
+	 * @param string $user
+	 * @param string $content
+	 * @param string $fileId
+	 *
+	 * @return void
+	 */
+	public function userHasUpdatedAFileWithContentUsingFileId(string $user, string $content, string $fileId): void {
+		$baseUrl = $this->getBaseUrl();
+		$sourceDavPath = WebdavHelper::getDavPath($this->getDavPathVersion());
+		$fullUrl = "$baseUrl/$sourceDavPath/$fileId";
+		$response = HttpRequestHelper::sendRequest(
+			$fullUrl,
+			$this->getStepLineRef(),
+			'PUT',
+			$user,
+			$this->getPasswordForUser($user),
+			null,
+			$content
+		);
+		$this->theHTTPStatusCodeShouldBe(['201', '204'], "Expected response status code doesn't match.", $response);
```

Comment:
the status code should always be `204` while updating the file
```suggestion
		$this->theHTTPStatusCodeShouldBe('204', '', $response);
```